test/cases/parsing/issue-2050/x.js

```diff
@@ -1,6 +1,6 @@
 import { xa, xb, xc, xd } from "./module";
 
-module.exports = {
+__webpack_module__.exports = {
```

Comment:
I am not sure what this test is trying to test, so I made it pass with the least amount of change.

Not sure if this should be `export * from "./module";`, `export { xa, xb, xc, xd } from "./module";`, or use `require("./module");` instead.